src/SDKs/Monitor/changelog.md

```diff
@@ -1,6 +1,41 @@
 ## Microsoft.Azure.Management.Monitor release notes
 
-### Changes in 0.17.0-preview
```

Comment:
This doc never showed the fact that 0.17.0-preview was removed from the public record, replaced with 0.18.0-preview.

My idea with this change is to reflect the changes in the past and give the changes for this release: 0.19.0-preview.